### DISCUSSION TITLE
文档

### DISCUSSION THREAD

**Discussion by 204065248** [netease-im/NIM_Duilib_Framework#283] [Q&A]
我看完了NIM_Duilib的 markdown文档，但是很多地方仍有不明白的地方，
比如下面这行代码的` align="center|vcenter" ` 是我自己懵出来的
` <Label width="80" height="35" text="属性检查器" align="center|vcenter" /> `
TabBox没有用法示例，xml应该怎么去写，XXXBox都说继承Box的属性，但是halign 等这个属性没看见哪里有提示，width等也没说都有什么特殊的值，比如auto。麻烦告知哪里有更详细的文档

**Comment by nmgwddj** [comment]
这里有一个非常完整的 IM Demo 程序，基于 NIM Duilib 构建，您可以参考这里 TabBox 的用法：https://github.com/netease-im/NIM_PC_Demo/search?q=tabbox

Box 是布局的基础能力实现，align 属性只需要填写一个你需要的值即可，文档中的确缺少示例程序，希望完整的代码示例可以帮助到你。

**Reply by 204065248** [reply]
好的- -，不知道有没有交流群呢，我找到网上的交流群都是基于原版的duilib 想询问NIM Duilib的问题 少有人回答

**Reply by nmgwddj** [reply]
交流群不利于信息保存，容易出现重复问题，这不是一个好的技术问题交流渠道。使用方面问题可以在 Discussions 讨论。发现的缺陷或新功能建议可以在 Issues 中提交。

**Reply by 204065248** [reply]
文档里的初始化项目的教程有点老了，我这些天自己看代码琢磨出来 创建项目的方法了，还是希望能够完善一下文档

**Reply by nmgwddj** [reply]
的确，目前 develop 分支合并了一些重要的需求，当这些需求完善后，我们准备发布 2.0 的版本，届时文档会同步更新。

**Reply by 204065248** [reply]
要有2.0了啊，期待啊，希望能够增加内存图片的加载方式，目前我看好像只有通过路径加载。不知道大概什么时候发布2.0呢

**Reply by xiaoyu21git** [reply]
你是怎么创建的可以分享一下吗

**Reply by 204065248** [reply]
> 你是怎么创建的可以分享一下吗

用啥给你分享
